### PR TITLE
Disable Facebook sync options for Virtual products

### PR DESCRIPTION
**Pull request by wvega**
This PR brings the following changes from the private repo:

```
 * Tweak - Hide Facebook options for virtual products and virtual variations
 * Tweak - Do not allow merchant to bulk enable sync for virtual products
 * Tweak - On upgrade, automatically disable sync for virtual products and virtual variations
```

### CODE REVIEW

File: class-wc-facebookcommerce.php [hunk 1]
```diff
@@ -108,6 +112,7 @@ public function init() {
 				$this->product_feed            = new \SkyVerge\WooCommerce\Facebook\Products\Feed();
 				$this->products_sync_handler   = new \SkyVerge\WooCommerce\Facebook\Products\Sync();
 				$this->sync_background_handler = new \SkyVerge\WooCommerce\Facebook\Products\Sync\Background();
+				$this->product_feed = new \SkyVerge\WooCommerce\Facebook\Products\Feed();
```
Comment (deivamagalhaes) [inline class-wc-facebookcommerce.php:115]: Duplicated.

File: includes/Lifecycle.php [hunk 2]
```diff
@@ -39,6 +39,7 @@ public function __construct( $plugin ) {
 			'1.10.0',
 			'1.10.1',
 			'1.11.0',
+			'2.0.0',
```
Comment (deivamagalhaes) [inline includes/Lifecycle.php:42]: I'd prefer to release this before `2.0.0`.

File: class-wc-facebookcommerce.php (hunk 1 above)
Comment (wvega) [reply class-wc-facebookcommerce.php]: Thank for catching that one. I removed the duplicated line in 6664e0d.

File: includes/Lifecycle.php (hunk 2 above)
Comment (wvega) [reply includes/Lifecycle.php]: I reverted the commits that change the version number with f6a1b29. We will need to revert others from the changes I merged yesterday, but we can do that in another PR after 1.11.3 is out.

**Comment by wvega** [comment]
Closing in favor of #1335. Everything change included in this PR is also there.

**Pull request closed without merging**